librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -35,6 +35,9 @@
 
 #include <rz_il/rz_il_opcodes.h>
 
+#undef AND
+#undef SIGNED
+
```

Comment:
these should go in the rz_il_opbuilder_end

Reply:
Okay